core/type/src/main/java/uk/gov/gchq/gaffer/types/TypeSubTypeValue.java

```diff
@@ -30,6 +32,7 @@
  * sub-types and associated values.
  */
 @JsonSimpleClassName
+@JsonTypeInfo(use = JsonTypeInfo.Id.CLASS, property = "class", defaultImpl = TypeSubTypeValue.class)
```

Comment:
Do you mean the `JSON` serialisation? The `JSON` serialisation stores a `HyperLogLogPlus` object so I don't it would be affected.  I have found that when the you create a `JSON` string with the `class` field added when the original object does not have the annotations then it ignores the field rather than throw an exception so this could be removed.

Reply:
I was just wondering if this was needed for json deserialisation, and if so, would it impact json serialisation? Basically, when I turn my TSTV into json, will it have class there whereas it didn't before?

Reply:
Yeah, not sure. As the annotation doesn't effect for the temporary fix it might be better to remove

Reply:
There are HistoricSerialisation tests that would break if this did change, also tested manually